<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="4407" started-at="2019-09-30T12:53:08Z" finished-at="2019-09-30T12:53:13Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="4407" started-at="2019-09-30T12:53:08Z" finished-at="2019-09-30T12:53:13Z">
      <class name="com.qa.tests.WeatherInfoTests">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.qa.tests.WeatherInfoTests@cb5822]" name="setup" is-config="true" duration-ms="5" started-at="2019-09-30T18:23:09Z" finished-at="2019-09-30T18:23:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="getweatherdetails(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.qa.tests.WeatherInfoTests@cb5822]" name="getweatherdetails" duration-ms="3194" started-at="2019-09-30T18:23:09Z" data-provider="getData" finished-at="2019-09-30T18:23:12Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Pune]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[GET]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[93 Percent]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[23.174 Degree celsius]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[3.05 Km per hour]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[257.188 Degree]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [23.174 Degree celsius] but found [23.513 Degree celsius]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [23.174 Degree celsius] but found [23.513 Degree celsius]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at com.qa.tests.WeatherInfoTests.getweatherdetails(WeatherInfoTests.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getweatherdetails -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.qa.tests.WeatherInfoTests@cb5822]" name="setup" is-config="true" duration-ms="1" started-at="2019-09-30T18:23:12Z" finished-at="2019-09-30T18:23:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="getweatherdetails(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.qa.tests.WeatherInfoTests@cb5822]" name="getweatherdetails" duration-ms="456" started-at="2019-09-30T18:23:12Z" data-provider="getData" finished-at="2019-09-30T18:23:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Patna]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[GET]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[94 Percent]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[25 Degree celsius]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[2.6 Km per hour]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[70 Degree]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [25 Degree celsius] but found [24 Degree celsius]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [25 Degree celsius] but found [24 Degree celsius]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at com.qa.tests.WeatherInfoTests.getweatherdetails(WeatherInfoTests.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getweatherdetails -->
      </class> <!-- com.qa.tests.WeatherInfoTests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
